src/external-site/_vars-external-site.html

```diff
@@ -6,7 +6,7 @@
    ========================================================================== */
 
     // Find the external-urls in your local cfgov-refesh repo.
-    find . -name '*.html' ! -path './src/*' ! -path './node_modules/*' ! -path './_tests/*' \
+    find . -name '*.html' ! -path './*' ! -path '../node_modules/*' ! -path '../test/*' \
```

Comment:
That is correct but this code can actually be removed at this point since we have the `.links.sh`. The code in `.link.sh` is accomplishing the same thing and is running as a Cron job of some sort.